### ISSUE TITLE
[BUG] ColorUI - Not possible to set PID parameters from LCD screen

### ISSUE THREAD

**Issue by bkiziuk** [MarlinFirmware/Marlin#20102]
### Bug Description

When trying to manually set PID-E parameters from LCD screen, entered values are not accepted.
Instead, following values are set:
PID-P E1: +001.0
PID-I E1: +000.01
PID-D E1: +001.0

### Configuration Files

[config.zip](https://github.com/MarlinFirmware/Marlin/files/5523890/config.zip)

### Steps to Reproduce

1. Set some value to PID-P E1, PID-I E1 or PID-D E1
2. Press Ok to accept value
3. Check the screen for actual value

**Expected behavior:**

The actual value should be equal to the accepted value.

**Actual behavior:**

Following values are used:
PID-P E1: +001.0
PID-I E1: +000.01
PID-D E1: +001.0

#### Additional Information

Board: MKS Robin Nano 1.2
Firmware: Marlin bugfix-2.0.x 2020-11-11

**Comment by rhapsodyv** [comment]
We had a bug with `SINGLE_TOUCH_NAVIGATION` that might be causing your issue. Can you test if this fixes your issue too? #20213

**Comment by bkiziuk** [comment]
Tested with bugfix-2.0.x 2020-11-20.
Issue is solved.

**Comment by boelle** [comment]
@bkiziuk if solved then click green close button below

**Comment by bkiziuk** [comment]
@boelle Thought that it needs to merged first ;-) But no problem here :-)

**Comment by sjasonsmith** [comment]
@bkiziuk merged into the bugfix branch yes. We don’t wait for them to be tagged in a versioned release to close them.
